shared/util/file.desktop.js

```diff
@@ -1,7 +1,6 @@
 // @flow
 import crypto from 'crypto'
 import fs from 'fs'
-import fsExtra from 'fs-extra'
```

Comment:
we had a fork of fs-extra cause it overrides methods in fs in a funky way which causes webpack to be unhappy. Also it was marked as a devDep even though we actually use it in real code...
Instead lets just use the builtin stuff